streaming/emr-eks-flink/examples/cluster-autoscaler/flink-sample-job.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: flink.apache.org/v1beta1
+kind: FlinkDeployment
+metadata:
+  name: basic-example-cluster-auto-flink
+  namespace: flink-team-a-ns
+spec:
+  imagePullPolicy: Always
+  emrReleaseLabel: "emr-7.0.0-flink-latest"
+  flinkVersion: v1_18
+  flinkConfiguration:
+    taskmanager.numberOfTaskSlots: "2"
+    # Autotuning parameters
+    kubernetes.operator.job.autoscaler.autotune.enable: "true"
+    kubernetes.operator.job.autoscaler.autotune.metrics.history.max.count: "2"
+    kubernetes.operator.job.autoscaler.autotune.metrics.restart.count: "1"
+    metrics.job.status.enable: TOTAL_TIME
+
+    # Autoscaler parameters
+    kubernetes.operator.job.autoscaler.enabled: "true"
+    kubernetes.operator.job.autoscaler.scaling.enabled: "true"
+    kubernetes.operator.job.autoscaler.stabilization.interval: "5s"
+    kubernetes.operator.job.autoscaler.metrics.window: "1m"
+
+    jobmanager.scheduler: adaptive
+    # Replace with s3 bucket in your own account
+    state.checkpoints.dir: s3://emr-flink-data/checkpoints
+    state.savepoints.dir: s3://emr-flink-data/savepoints
```

Comment:
This should be replaced with the S3 bucket output created by this terraform deployment. You can write these instructions on the top of the file as a NOTE.